utils/nctl/sh/scenarios/itst01.sh

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env bash
+
+source "$NCTL"/sh/utils/main.sh
+source "$NCTL"/sh/views/utils.sh
+source "$NCTL"/sh/node/svc_"$NCTL_DAEMON_TYPE".sh
+source "$NCTL"/sh/utils/infra.sh
+
+# Exit if any of the commands fail.
+set -e
+
+#######################################
+# Runs an integration tests that tries to simulate
+# stopping and restarting a random node.
+#
+# Arguments:
+#   `timeout=XXX` timeout (in seconds) when syncing.
+#######################################
+function main() {
+    log "------------------------------------------------------------"
+    log "Starting Scenario: itst01"
+    log "------------------------------------------------------------"
+
+    log "Node to be stopped: $NODE_ID"
+
+    # 0. Wait for network start up
+    do_await_genesis_era_to_complete
+    # 1. Allow chain to progress
+    do_await_era_change
+    # 2. Stop random node
+    do_stop_node "$NODE_ID"
```

Comment:
You're passing in the `NODE-ID` here to stop but this node is implicitly also started in `do_start_node` (i.e. we don't pass in the node id) – let's make it consistent across the whole test.

Reply:
fixed